File: lab/javascript/matematica_e_operadores/quadrado_de_um_numero.js
function quadrado(n){
    return Math.pow(n, 2);
}


function quadrado2(n){
    return n**2;
}


function quadrado3(n){
    return n * n;
}


const prompt = require('prompt-sync')();
let n = Number(prompt());
console.log(quadrado(n));
console.log(quadrado2(n));
console.log(quadrado3(n));

File: lab/python/matematica_e_operadores/quadrado_de_um_numero.py
import math


def quadrado(n):
    return math.pow(n, 2)


def quadrado2(n):
    return n**2


def quadrado3(n):
    return n * n


n = int(input())
print(quadrado(n))
print(quadrado2(n))
print(quadrado3(n))
